draft-ietf-quic-tls.md

```diff
@@ -1743,6 +1186,10 @@ many others.
 
 Issue and pull request numbers are listed with a leading octothorp.
 
+## Since draft-ietf-quic-tls-11
+
+- Remove source address validation from TLS.
```

Comment:
That's not really the only thing that changed in this PR.

Reply:
Fixed